.github/ISSUE_TEMPLATE/issue-report.md

```diff
@@ -0,0 +1,44 @@
+---
+name: Issue report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Before opening an issue**
+
+Questions should go to the GEE mailing list (https://groups.google.com/forum/?utm_medium=email&utm_source=footer#!forum/gee-oss)
+or Slack Open GEE channel (http://slack.opengee.org/) GitHub issues are for bug reports and suggestions for new features.
+
+**Describe the bug**
+
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+
+Steps to reproduce the behavior:
+1. Start Open GEE 5.2.5
```

Comment:
I think having an example here is a little confusing and I suspect people will leave the example instead of replacing it. I say we leave it blank for now and add one if people are still getting confused.

Reply:
Edited